.npmignore

```diff
@@ -1,3 +1,4 @@
+.rpt2_cache/
```

Comment:
We can also add `ISSUE_TEMPLATE.md` and `pull_request_template.md` to `.npmignore` as they are Github related docs. We can also add `rollup.config.js` since it is part of build system.